react/store/createReturnRequest/components/ItemsDetails.tsx

```diff
@@ -24,6 +24,20 @@ const CSS_HANDLES = [
   'productImageWrapper',
   'productImage',
   'itemsDetailText',
+  'cardWrapper',
+  'productImageWrapper',
+  'productImage',
```

Comment:
These two classes are duplicated.
If we are intending to have a different use for them, can we rename them, and if not, remove it?

Reply:
Hi @edyespinal  could you please validate it again?